<launch>
    <node pkg= "traffic_detection" name="detect_darknet" type="darknet_detect_ros.py"  output="screen"> 
    </node>

        
    <node pkg="rosbag" type="play" name="rosbag" args="-d 1 /home/autodrive/BagFiles/2022-03-01/2022-03-01-17-58-18.bag">

        <!-- <param name="image_width" value="600" />
        <param name="image_height" value="600" /> -->
        <!-- <remap from="" to="/odem_in"/> -->
        <!-- <remap from="/resize_img/image" to="/image_in"/> -->
    </node>

    <!-- <node pkg="image_view" type="image_view" respawn="false"  name="raw_feed" output="screen">
        <remap from="image" to="/front_camera/image_raw" />
    </node> -->

    <node name="rqt_image_view" pkg="rqt_image_view" type="rqt_image_view" args="/visualize_image"/>

</launch>